reactive/src/main/java/com/scmspain/howtospring/reactive/Application.java

```diff
@@ -6,13 +6,12 @@
 import org.springframework.boot.autoconfigure.SpringBootApplication;
 import org.springframework.cloud.client.SpringCloudApplication;
 import org.springframework.cloud.netflix.eureka.EnableEurekaClient;
-import org.springframework.cloud.netflix.rx.RxJavaAutoConfiguration;
 import org.springframework.context.annotation.Import;
 import org.springframework.scheduling.annotation.EnableAsync;
 
-@EnableEurekaClient()
-@SpringBootApplication(exclude = RxJavaAutoConfiguration.class)
-@SpringCloudApplication()
+@EnableEurekaClient
+@SpringBootApplication
+@SpringCloudApplication
```

Comment:
Actually, we could remove @EnableEurekaClient too, @EnableDiscoveryClient is included by SpringCloudApplication too.

In ms-vibbo--user-rating, we only use: @SpringCloudApplication. Even @EnableAsync I think is not necessary.

Despite it's not the target of this PR, I'll remove these lines. I only wanted to remove RxJavaAutoConfig exclude statement.